### ISSUE TITLE
{Design] Join queue page on mobile

### ISSUE THREAD

**Issue by maaverik** [SimplQ/simplQ-frontend#331]
![Screenshot from 2020-10-04 16-27-18](https://user-images.githubusercontent.com/12583528/95013559-ab908600-065e-11eb-87f2-820fd73a725e.png)

The join page isn't really suitable for mobile view right now. If we just use a flex, the form would either show up before the header or after the details (which would make the user have to scroll down to join) @akashkbaburajan could you think of a suitable mobile view for this?

**Comment by daltonfury42** [comment]
We should use the same general layout of the status page if possible. It will give the user a consistent feel and also we will be able to reuse a lot of code

https://xd.adobe.com/view/ad1db074-03bf-45b1-537b-98d9d524ec82-db2c/screen/e87838cd-febc-4f37-a68a-9daa0d6b207b/specs/

**Comment by akashkbaburajan** [comment]
yes, I have redesigned the page and updated the design link. How does this look
![Adobe XD 04-10-2020 19_42_00](https://user-images.githubusercontent.com/16539463/95018016-6bd79780-067a-11eb-8b13-1c62d0764dea.png)

**Comment by daltonfury42** [comment]
I'm liking this

**Comment by maaverik** [comment]
Let's use this design and make a new issue to create this page? The header is being worked on anyway right now in #348. @akashkbaburajan can you provide the XD link for this page?

**Comment by daltonfury42** [comment]
I'd suggest splitting to three tasks:

Task 1: Refactor header to common folder and then just use it in the join page.
Task 2: Style left Pane
Task 3: Style right pane

Where task 1 is blocked on #348 and task 2 and 3 are "You can do this". But. I know you don't like splitting to subtasks :)

**Comment by maaverik** [comment]
Task 2 is pretty small, nothing much to be done there. Task 3 would take more work and maybe an extra API call. Could you make an issue on task 3 detailing the API which would need to be used? I guess the same one as in QueueDetails. As for splitting, I don't mind either way. Whoever's doing task 3 can do task 2 easily. I'll leave that up to you :).

**Comment by daltonfury42** [comment]
Can I do it in the weekend?

Another thought: We are using a 3rd party phone number input and material UI's text box. Changing their styling might be pain. Can we stick to the input boxes that we have, for now?

**Comment by maaverik** [comment]
The thing that's hard to replace about the materialUI phone input is the flag that comes builtin. I personally like it a lot, and it makes it obvious that you're expected to enter the right country code. So my opinion is to keep it at least until we face said styling issue or find a good alternative.

**Comment by daltonfury42** [comment]
right. It's a different library than material UI that Sajmal found. It's working well for us: https://www.npmjs.com/package/react-phone-input-2

We are using their material ui theme. So let's keep the input boxes as is (material like) and implement the rest.
